src/pages/Marketplace/Cards/index.tsx

```diff
@@ -29,9 +29,11 @@ export default function Cards({ classes = "" }: { classes?: string }) {
               : "grid-cols-[repeat(auto-fit,minmax(245px,1fr))]"
           } gap-4 content-start`}
         >
-          {endowments.map((endow) => (
-            <Card {...endow} key={endow.id} />
-          ))}
+          {endowments
+            .filter((endowment) => endowment.published)
```

Comment:
@jp-mariano, do we have filter for `published:boolean` for `/v3/endowments/${network}` ? - filtering client side isn't accurate because it doesn't consider next pages

Reply:
> @jp-mariano, do we have filter for `published:boolean` for `/v3/endowments/${network}` ? - filtering client side isn't accurate because it doesn't consider next pages

Yep, this is already implemented. The link to the `devops` PR is [here](https://github.com/AngelProtocolFinance/devops/pull/206).